templates.tf

```diff
@@ -13,6 +13,7 @@ data "template_file" "shuntingyard_config_yaml" {
     shuntingyard_sqs_queue              = "${aws_sqs_queue.shuntingyard_sqs_queue.id}"
     shuntingyard_sqs_queue_wait_timeout = "${var.shuntingyard_sqs_queue_wait_timeout}"
     selected_tables                     = "${join("\n", formatlist("    - %s", var.selected_tables))}"
+    orphaned_data_strategy              = "${format("orphaned-data-strategy: %s", var.orphaned_data_strategy)}"
```

Comment:
Should we call this `shunting_yard_orphaned_data_strategy` since it applies specifically to shunting yard?

Reply:
So this is the same `orphaned_data_strategy` that is used by Circus Train. Shunting yard uses this parameter for some business logic (e.g. deciding what CT modules to run) but SY also feeds it directly into the CT config:

https://github.com/ExpediaGroup/shunting-yard/blob/3eb98c676b224f0efc36f15b01243ffcb8370fbd/shunting-yard-replicator/src/main/java/com/expediagroup/shuntingyard/replicator/exec/receiver/ContextFactory.java#L125

So really it is the same parameter as used in Circus Train. Also, renaming it here would require that we re-release SY.

Reply:
OK, fine, let's leave it as is then.